servers/src/test/java/tachyon/master/next/filesystem/meta/InodeDirectoryTests.java

```diff
@@ -104,9 +104,11 @@ public void removeNonExistentChildTest() {
   @Test
   public void deleteInodeTest() {
     InodeDirectory inode1 = createInodeDirectory();
-    Assert.assertEquals(false, inode1.isDeleted());
+    Assert.assertFalse(inode1.isDeleted());
     inode1.delete();
-    Assert.assertEquals(true, inode1.isDeleted());
+    Assert.assertTrue(inode1.isDeleted());
+    inode1.restore();
```

Comment:
This is because when I checked out the branch, the branch was in an inconsistent state. And the changes in this file were to resolve inconsistency.